manuscript/060_Statement_First_Programming.md

```diff
@@ -135,39 +134,41 @@ var configuration = new XmlConfiguration(customFixture);
+And it passes -- cool... not. Ok, what is wrong with this? Nothing big, unless you read the source code of XmlConfiguration class carefully. Inside, you can see, how the XML string is stored:
 
 ```csharp
 private static string xmlText; //note the static keyword!
 ```
 
-What the...? Well, well, here is what happened: the author of this class coded in a�small little optimization. He thought: �In this app, the configuration is only modified by members of the support staff and to do it, they have to shut down the system, so, there is no need to read the XML file every time an XmlConfiguration object is created. I�can save some CPU cycles and I/O operations by reading it only once when the first object is created. Another created object will just use the same XML!". Good for him, not so good for you. Why? Because (unless your Statement is evaluated prior to being fulfilled), your custom xml string will never actually be used!
+What the...? Well, well, here is what happened: the author of this class applied a small optimization. He thought: “In this app, the configuration is only modified by members of the support staff and to do it, they have to shut down the system, so, there is no need to read the XML file every time an XmlConfiguration object is created. I can save some CPU cycles and I/O operations by reading it only once when the first object is created. Later objects will just use the same XML!". Good for him, not so good for you. Why? Because your custom XML string will never actually be used! (Unless your Statement is evaluated prior to being fulfilled.)
 
-�Test-After" ends up as �Test-Never" 
+“Test-After" ends up as “Test-Never" 
 ------------------------------------
 
-I will ask this question again: ever had to write a�requirement or design document for something that you already implemented? Was it fun? Was it valuable? Was it creative? No, I�do not think so. The same is with our executable specification. After we write the code, we have little motivation to specify what is already written -- some of the pieces of code �we can just see are correct", other pieces �we already saw working" when we copied our code over to our deployment machine and ran few sanity checks... The design is ready... Specification? Maybe next time...
+I will ask this question again: did you ever have to write a requirements or design document for something that you already implemented? Was it fun? Was it valuable? Was it creative? No, I do not think so. The same holds for our executable specification. After we've written the code, we have little motivation to specify what we wrote -- some of the pieces of code “we can just see are correct", other pieces “we already saw working" when we copied the code over to our development machine and ran a few sanity checks... The design is ready... Specification? Maybe next time...
```

Comment:
Ok, I think I see what made you confused.

"other pieces “we already saw working" when we copied the code over to our deployment machine and ran a few sanity checks"

means:

"other pieces “we already saw working" when we compiled and deployed our changes and ran a few manual checks". I don't know how to express it better. This means building the product with the changes, deploying it on any environment (usually a test environment) and checking the logic by running few simple manual tests.